src/main/steps/tasklistresponse/proceedings/ProceedingDetailsMapper.ts

```diff
@@ -162,6 +162,9 @@ function getLocalDate(orderDate: string): Date {
   if (orderDate['year'] === '' && orderDate['month'] === '' && orderDate['day'] === '') {
     const formated_Date = new Date(orderDate[''], orderDate[''], orderDate['']);
     return formated_Date;
+  } else if (orderDate) {
+    const formated_Date = new Date(orderDate['year'], orderDate['month'] - 1, orderDate['day']);
```

Comment:
why need a variable unnecessarily when it is returned in the next line. Refactor the entire method.

Reply:
Yeah directly returning the value now.. instead of creating that variable.